cli/src/mesh.rs

```diff
@@ -43,7 +43,7 @@ pub async fn monitor_mesh() -> anyhow::Result<()> {
         if let Some(addr) = depart_rx.recv().await {
             println!("❌ {} {}", "DEPARTED:".red(), addr);
         }
-        sleep(Duration::from_secs(5)).await;
+        sleep(Duration::from_secs(1)).await;
```

Comment:
I think we can probably roughly copy this pattern from `KaboodleInner::tick` to avoid the timeout entirely:

````rs
loop {
                tokio::select! {
                    _ = self.broadcast_in_sock.readable() => self.handle_incoming_broadcasts().await,
                    _ = self.sock.readable() => self.handle_incoming_messages().await,
                };
            }
````